src/main/java/frc/robot/commands/AutonBalance.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.subsystems.DriveSubsystem;
+
+public class AutonBalance extends SequentialCommandGroup {
+
+  private class Balance extends PIDCommand {
+    DriveSubsystem driveSubsystem;
+
+    public Balance(DriveSubsystem driveSubsystem) {
+      super(new PIDController(0.3, 0, 0), driveSubsystem.pigeon2::getRoll, 0, output -> {
```

Comment:
I was getting funky readings from the Gyro yesterday. Jacky adjusted the wire for the gyro, so this should be retested before merging due to the aforementioned gyro issues.

Reply:
Unclear if we are reading roll or pitch as `isFinished` is still checking for pitch. I'm going to assume that pitch is the correct measurement for now.